docs/customizing_dashboards.md

```diff
@@ -76,4 +76,12 @@ if the value is stored by the number of cents:
   )
 ```
 
+Or if you want to apply a different datetime format:
+```ruby
+  date_of_birth: Field::DateTime.with_options(
+    title: "Birthday",
+    format: "%-d %B",
```

Comment:
Is there an out of the box format style we could use that Rails already defines?